# df_paw parameters

paw_port : /dev/fumi_motors
baud_rate : 115200
paw_frame : base_link
paw_left_id: 2
paw_right_id: 1

cmd_topic: cmd_vel
cb_timeout : 0.5
claw_addr : 128
allow_cmd_timeout : true
gear_ratio: 1/5


#common parameters
speed_topic : wheelspeed
wheel_radius : 0.0625
wheel_base : 0.2
loop_rate  : 15

#odom parameters
odom_topic : wheelodom
broadcast_odom_tf : false
speed_cb_timeout : 0.5
